IBAnimatableTests/RotationDesignableTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  RotationDesignableTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/30/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+final class RotationDesignableTests: XCTestCase {
+
+  override func setUp() {
+    super.setUp()
+  }
+
+  override func tearDown() {
+    super.tearDown()
+  }
+
+  func testRotate() {
+    helper_testRotate(element: AnimatableView())
+    helper_testRotate(element: AnimatableScrollView())
+    helper_testRotate(element: AnimatableImageView())
+    helper_testRotate(element: AnimatableStackView())
+    helper_testRotate(element: AnimatableLabel())
+    helper_testRotate(element: AnimatableSlider())
+  }
+
+  private func helper_testRotate(element: MockRotationDesignable) {
+    element.rotate = -360
+    XCTAssertEqual(element.transform, .identity)
+    element.rotate = 360
+    XCTAssertEqual(element.transform, .identity)
+    element.rotate = 90
+    let mockTransform = CGAffineTransform(rotationAngle: .pi * 90 / 180)
+    XCTAssertEqual(element.transform, mockTransform)
+  }
+}
+
+// MARK: - MockRotationDesignable
+
+protocol MockRotationDesignable: class, RotationDesignable {
```

Comment:
I'm not sure what you mean by creating a file by mock. Right now I'm testing each protocol and creating a mock for that protocol. One protocol per file.

As for testing mixing designable properties... basically, you're talking about one designable protocol affecting the performance of another designable protocol? That's an interesting point you bring up. I'm just testing the individual components right now... mixing them does add another degree of complexity.